core/src/test/java/io/questdb/test/cairo/mv/MatViewFuzzTest.java

```diff
@@ -204,6 +204,18 @@ public void testManyTablesView() throws Exception {
         });
     }
 
+    @Test
+    public void testMultipleQueryExecutionsPerRefresh() throws Exception {
+        final Rnd rnd = fuzzer.generateRandom(LOG);
+        final int rowsPerQuery = Math.max(1000, rnd.nextInt(500_000));
```

Comment:
Please set the lower limit so that it is exercised in some tests

Reply:
Addressed in 8d17c21d7ccd2f9784042da87b26aeed39f4fe4f